pkg/download/classic/downloader.go

```diff
@@ -49,83 +49,37 @@ type Downloader struct {
 	client dtclient.Client
 }
 
-// WithAPIContentFilters sets the api content filters for the Downloader - these will be used in addition to base API filtering
-func WithAPIContentFilters(apiFilters map[string]contentFilter) func(*Downloader) {
-	return func(d *Downloader) {
-		d.apiContentFilters = apiFilters
-	}
-}
-
-func WithAPIs(apis api.APIs) func(*Downloader) {
-	return func(d *Downloader) {
-		d.apisToDownload = apis
-	}
-}
-
-// NewDownloader creates a new Downloader
-func NewDownloader(client dtclient.Client, opts ...func(*Downloader)) *Downloader {
+// NewDownloader creates a new sound Downloader.
+func NewDownloader(client dtclient.Client, opts ...func(*Downloader) error) (*Downloader, error) {
 	c := &Downloader{
-		apisToDownload:    api.NewAPIs(),
```

Comment:
shouldn't the default still be `api.NewAPIs()`?
As `WithEndpoints` is optional, the current default initialized downloader has no APIs to download at all.

Reply:
This doesn't have an impact on our code. It is more how we would like to design input for the library. 
I'll make `api.NewAPIs()` to be default